src/Admin/AdministratorFactory.php

```diff
@@ -14,6 +14,10 @@ public function __construct(EncoderFactoryInterface $encoders)
         $this->encoders = $encoders;
     }
 
+    /**
+     * @param array $data
+     * @return Administrator
+     */
```

Comment:
This docblock is not really needed since it does not bring any information.
Type hints explain it all.

Reply:
It's just to ask what is array? what should it contains?

Reply:
Then you should add a description to the argument:

/**
 * @param array $data The collection indexed by name.
 */

Only then the docblock brings an extra information, but is it useful ?